FeBuddyWinFormUI/Program.cs

```diff
@@ -14,14 +14,27 @@ static class Program
         [STAThread]
         static void Main()
         {
-            // Create a shortcut on initial install using Squirrel
-            SquirrelAwareApp.HandleEvents((onInitialInstall) => new UpdateManager($"{GlobalConfig.tempPath}", "FE-Buddy").CreateShortcutForThisExe(ShortcutLocation.StartMenu | ShortcutLocation.Desktop));
+            
+            
 
             // TODO - Get system info and log it into file first thing. -https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ee436483(v=msdn.10)
             Logger.CreateLogFile();
             Logger.LogMessage("DEBUG", "PROGRAM STARTED");
 
-            //Console.WriteLine(Logger.logFilePath);
+            // Remove the Start Menu shortcut in the Kyle Sanders directory and replace with one in the root on initial install and update
+            SquirrelAwareApp.HandleEvents(onAppUpdate: (v) =>
+            {
+                new UpdateManager($"{GlobalConfig.tempPath}", "FE-Buddy").CreateShortcutForThisExe(ShortcutLocation.StartMenuRoot | ShortcutLocation.Desktop);
```

Comment:
@wiggleforlife - When you get a moment will you address this? Thank you 

@caesay Thank you for your help and comments.

Reply:
Sorry, I'm not totally familiar with C#, coming from Java. Is the `using var` the best way of going about this or shall I use a different method, @caesay ?
Sorry as well for the long delay.

Reply:
You can use `using var localName =` and it will be kept alive until the end of the method, or you can use `using (var localName =) { ... }` and use the curly brackets to be explicit about when the object is disposed.